deploy/olm-catalog/ibm-block-csi-operator/1.7.0/bundle-1.7.0.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM scratch
+
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=ibm-block-csi-operator
+
+COPY manifests /manifests/
+COPY metadata /metadata/
+LABEL com.redhat.openshift.versions="v4.8,v4.9"
```

Comment:
really? is that confirmed with eran?

Reply:
it's our default currency (and "leaving" it as "v4.7,v4.8" is not "confirmed" either).
if we ever decide to diverge from our current currency, we would need a separate ticket (otherwise - we're good)

Reply:
why not just confirmed it before the change?

Reply:
the final decision whether to change or not might take awhile, we want to have csv yamls until then

Reply:
so let's do it after the decision to do it and in the meanwhile leave it with 4.7,4.8?

Reply:
@erantzabari , what say you?

Reply:
Arbel, we spoke on it last week.. the plan for 4.9 is September - so we are aiming to support it.
If for some reason we won't be able - we will consider removing it. Though the mindset is currency.

Reply:
ok, no problem